conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import ConanFile, CMake
+
+
+class Libgit2Conan(ConanFile):
+    name = "libgit2"
+    version = "0.28.0"
```

Comment:
If we were to check in packaging recipes, then I'd really prefer to not include any changing bits like version numbers. There's quite a lot of different packaging systems, and having to update bits like these in all of them becomes a major burden on us, as we needn't forget to do so on every release.

Instead, I'd propose to parse the version number from our central source-of-truth version number, which is located in "include/git2/version.h".

Reply:
I agree. I'm still making the MVP, I'll update this after MVP is done.